src/Resources/ResourceManager/SdkModels/Deployments/DeploymentVariable.cs

```diff
@@ -23,5 +29,17 @@ public class DeploymentVariable
 
         [JsonProperty("value")]
         public object Value { get; set; }
+
+        public override string ToString()
```

Comment:
Please use a name that better describes its function, such as ToFormattedString()

Reply:
Hi @msJinLei , thanks for your comments. I think ToString() is good here as its ToString() only show its type before. No breaking change will introduce here. And I hope user can obtain formatted string anytime.
![image](https://user-images.githubusercontent.com/19869716/145922588-e1fb41f9-301d-4daa-8eaa-dcd7136e9528.png)

Shall we keep ToString() as the function name?

Reply:
> Hi @msJinLei , thanks for your comments. I think ToString() is good here as its ToString() only show its type before. No breaking change will introduce here. And I hope user can obtain formatted string anytime. ![image](https://user-images.githubusercontent.com/19869716/145922588-e1fb41f9-301d-4daa-8eaa-dcd7136e9528.png)
> 
> Shall we keep ToString() as the function name?

The function contains the logical to control its display format rather serialize it to string. It is better to have those who use it realize it by its name.